pyart/retrieve/comp_z.py

```diff
@@ -0,0 +1,109 @@
+"""
+Calculate the composite reflectivity 
+
+"""
+
+import copy
+import numpy as np
+from netCDF4 import num2date
+from pandas import to_datetime
+
+def composite_reflectivity(radar,rhv_filter=True,rhv_value=0.95,verbose=False):
+    
+    """
+    Composite Reflectivity 
+    
+    Often a legacy product, composite reflectivity is: 
+    "A display or mapping of the maximum radar reflectivity factor at any altitude as a function of position on the ground." - AMS Glossary 
+    
+    This is more useful for the dry regions of the world, where maximum reflectivity values are found aloft, as opposed to the lowest
+    scan. Alternatively this is useful for comparing to NWP since composite Z is a standard output of NWP. 
+
+
+    Why this is not as easy as one would think: Turns out the data are not natively stored with index 0 being azimuth 0. Likely due
+    to the physical spinning of the radar antenna. 
+    
+    Author: Randy J. Chase (@dopplerchase) 
+    
+    Note: this function could potenially be parallelized at the loop, but for now runs in ~5 seconds on my 16 GB Mac
+    
+    Parameters
+    ----------
+    radar : Radar
+        Radar object used.
+    rhv_filter : bool
+        boolean flag to turn on correlation coef. filtering. 
+    rhv_value : float
+        value to consider 'meteorological'
+    verbose : bool
+        boolean flag to turn the printing of radar tilts on or off. 
+
+    Returns
+    -------
+    comp_dict : Dictonary
+        A dictionary containing the sorted lon,lat grid and the composite reflectivity value
+        This will be [azimuth,range]
+        
+    """
+    
+    #loop over all measured sweeps 
+    for sweep in radar.sweep_number['data']:
+    
+        #get start and stop index numbers
+        s_idx = radar.sweep_start_ray_index['data'][sweep]
+        e_idx = radar.sweep_end_ray_index['data'][sweep]+1
+        
+        #grab radar data 
+        z = radar.get_field(sweep,'reflectivity')
```

Comment:
Make sure to add a `reflectivity_field` argument so the user can specify this in the function